src/otx/recipe/classification/multi_class_cls/dino_v2.yaml

```diff
@@ -1,22 +1,22 @@
 model:
-  class_path: otx.algo.classification.dino_v2.DINOv2RegisterClassifier
+  class_path: otx.algo.classification.vit.VisionTransformerForMulticlassCls
   init_args:
     label_info: 1000
-    freeze_backbone: False
-    backbone: dinov2_vits14_reg
+    arch: "dinov2-small"
 
     optimizer:
       class_path: torch.optim.AdamW
       init_args:
-        lr: 1e-5
+        lr: 0.0001
+        weight_decay: 0.05
 
     scheduler:
       class_path: lightning.pytorch.cli.ReduceLROnPlateau
       init_args:
-        mode: min
+        mode: max
         factor: 0.5
-        patience: 9
-        monitor: train/loss
+        patience: 1
```

Comment:
Why patience is so small? I think patience = 1 doesn't suit any training pipelines since the metric can fluctuate during training.

Reply:
I have just followed other multiclass-cls recipes.
Please refer to 
https://github.com/openvinotoolkit/training_extensions/blob/9b590823e24775129731ed8a4ea3e8931bc4fc59/src/otx/recipe/classification/multi_class_cls/efficientnet_b0.yaml#L19
https://github.com/openvinotoolkit/training_extensions/blob/9b590823e24775129731ed8a4ea3e8931bc4fc59/src/otx/recipe/classification/multi_class_cls/deit_tiny.yaml#L19